ci.go

```diff
@@ -0,0 +1,189 @@
+package main
+
+import (
+	"github.com/mysteriumnetwork/winter-server/core"
+	"os"
+	"fmt"
+	"strings"
+	"runtime"
+)
+
+const IS_WINDOWS = runtime.GOOS == "windows"
+
+func build(binaryName, packageName string) {
+	if IS_WINDOWS {
+		binaryName += ".exe"
+	}
+	core.MustRunCommand("go", "build", "-i", "-o", "./out/" +binaryName, packageName)
+	fmt.Printf("Build successful: out/%s\n", binaryName)
+}
+
+func buildServer() {
+	build("server", "./server")
+}
+
+func buildServerImage() {
+	buildDocker("docker/server/Dockerfile", "mysteriumnetwork/winter-server")
+}
+
+func releaseServerImage(versions []string) {
+	publishDocker("mysteriumnetwork/winter-server", "mysteriumnetwork/winter-server", versions)
+}
+
+func buildClient() {
+	build("client", "./client")
+}
+
+func buildWeb() {
+	build("web", "./client_web")
+	if IS_WINDOWS {
+		core.Execute("xcopy /f /y client_web\\html out\\html")
+	} else {
+		core.Execute("cp -a ./client_web/html ./out")
+	}
+}
+
+func buildDocker(dockerfile, image string) {
+	fmt.Printf("Building image '%s'..\n", image)
+
+	core.MustRunCommand("docker", "build", "--file", dockerfile, "--tag", image, ".")
+	fmt.Print("Docker image building process complete!")
+}
+
+func publishDocker(sourceImage, targetImage string, targetVersions []string) {
+	for _, version := range targetVersions {
+		targetImageWithVersion := fmt.Sprintf("%s:%s", targetImage, version)
+		fmt.Printf("Publishing image version '%s'..\n", targetImageWithVersion)
+
+		core.MustRunCommand("docker", "tag", sourceImage, targetImageWithVersion)
+		core.MustRunCommand("docker","push", targetImageWithVersion)
+	}
+}
+
+func doRun(args []string) {
+	help := `
+Usage: 
+	go run ci.go run <artifact>
+
+The artifacts are:
+	server    builds and runs server app
+	client    builds and runs client app
+	web       builds and runs client app with integrated website
+	help
+`
+
+	if len(args) != 1 {
+		fmt.Println("Need <artifact> as first argument")
+		fmt.Print(help)
+		os.Exit(1)
+	}
+
+	switch strings.ToLower(args[0]) {
+	case "server":
+		buildServer()
+		core.ExecuteIn("./server", "./out")
+	case "client":
+		buildClient()
+		core.ExecuteIn("./client", "./out")
+	case "web":
+		buildWeb()
+		core.ExecuteIn("./web", "./out")
+	case "help":
+		fmt.Print(help)
+	default:
```

Comment:
Should the `doRun` have the default option that only builds everything and run nothing?